txnkv/transaction/2pc.go

```diff
@@ -1401,6 +1412,9 @@ func (c *twoPhaseCommitter) execute(ctx context.Context) (err error) {
 		logutil.Logger(ctx).Debug("2PC will use async commit protocol to commit this txn",
 			zap.Uint64("startTS", c.startTS), zap.Uint64("commitTS", c.commitTS),
 			zap.Uint64("sessionID", c.sessionID))
+		if c.store.IsClose() {
+			return tikverr.ErrStoreIsExiting
```

Comment:
Similar to committing secondary locks, we shouldn't return an error here. Otherwise, the user may receive commit error but the transaction succeeds to commit.

Reply:
Done